src/routes/news/containers/LocalNewsDetailsPage.js

```diff
@@ -15,15 +15,18 @@ type PropsType = {|
   language: string,
   city: string,
   path: string,
+  cities: Array<CityModel>,
   t: TFunction
 |}
 
 export class LocalNewsDetailsPage extends React.PureComponent<PropsType> {
   render () {
-    const { localNewsElement, language, city, path } = this.props
+    const { localNewsElement, language, city, path, cities } = this.props
 
-    if (!localNewsElement) {
-      const error = new ContentNotFoundError({ type: 'localNewsItem', id: path, city, language })
+    const currentCity: CityModel = cities && cities.find(cityElement => cityElement.code === city)
+    if (!localNewsElement || !currentCity.pushNotificationsEnabled) {
+      const type: string = currentCity.pushNotificationsEnabled ? 'localNewsItem' : 'category'
+      const error = new ContentNotFoundError({ type, id: path, city, language })
       return <FailureSwitcher error={error} />
     }
```

Comment:
I'd rather split in if else if like in TunewsDetailsPage. But your choice.

Reply:
Done.